### PR TITLE
hw-1

### PR DESCRIPTION
**Pull request by M1lya1**

### CODE REVIEW

File: students/M1lya1/hw-1/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <div></div>  
+    <button>click</button>
+    <script src="./script.js"></script>
+    <script>
+        setTimeout(() => {
+            alert('click on button');
+        }, 3500)
+    </script>
+</body>
+</html>
```
Comment (Temu4) [inline students/M1lya1/hw-1/index.html:19]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/M1lya1/hw-1/script.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+document.querySelector('button').addEventListener('click', () => {
+
+    const div = document.querySelector('div');
+
+    div.innerHTML = `hello student`;
```
Comment (Temu4) [inline students/M1lya1/hw-1/script.js:5]: you don't need Template string here. Consider to replace it to `'` or `"`

File: students/M1lya1/hw-1/script.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+})
```
Comment (Temu4) [inline students/M1lya1/hw-1/script.js:6]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

**Review by Temu4 — CHANGES_REQUESTED** [review]
Good job! 🐱 
But still a few issues ->

**Comment by M1lya1** [comment]
i'm added a new line, all changes in new commit

**Pull request merged**